appinfo/info.xml

```diff
@@ -5,7 +5,7 @@
 	<name>ShareImporter</name>
 	<summary>Import Shares in Nextcloud app</summary>
 	<description>Import Shares in Nextcloud app</description>
-	<version>1.0.4</version>
+	<version>1.0.6</version>
```

Comment:
direct jump to 6 ?

Reply:
I pushed an empty 1.0.5 tag by mistake, so I need to use 1.0.6 instead :(

It doesn't really cause any problem for the app itself if we jump a version, so it's not a big issue